### DISCUSSION TITLE
Welk licentie model kiezen we?

### DISCUSSION THREAD

**Discussion by rubenvdlinde** [OpenCatalogi/.github#27] [Q&A] [answered]
In commonground wordt doorgaans gebruik gemaakt van EUPL

**Comment by rubenvdlinde** [comment]
EUPL staat overigens ook gewoon in de leidraad commonground https://www.gemmaonline.nl/images/gemmaonline/8/83/Informatiekundige_visie_Common_Ground_v0.9.pdf

**Comment by RvdKlip** [comment, marked as answer]
Wat mij betreft gewoon EUPL

https://joinup.ec.europa.eu/collection/eupl/impact-eupl
